.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,91 @@
+name: 🐛 Bug Report
+description: Report a bug encountered while using Cluster API
+labels: kind/bug
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What steps did you take and what happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please follow the guidelines described in https://github.com/kubernetes-sigs/cluster-api/blob/main/SECURITY_CONTACTS
+      placeholder: "[A clear and concise description on how to REPRODUCE the bug.]"
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?
+    validations:
+      required: false
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Anything else you would like to add?
+      placeholder: "[Miscellaneous information that will assist in solving the issue.]"
+
+  - type: textarea
+    id: capiVersion
+    attributes:
+      label: Cluster-api version
+      value: |
+        <details>
+
+        </details>
+    validations:
+      required: true
+
+  - type: textarea
+    id: kubeVersion
+    attributes:
+      label: Kubernetes version
```

Comment:
wondering if we really need this in the form...
Also, sometimes Kubernetes version is hard to interpret in a CAPI context. is it the Kubernetes version of the kind cluster,  Kubernetes version of the management cluster version or Kubernetes version a workload cluster?
I'm kind of leaning toward keeping the form simple... opinions? 
(same applies to Minikube/Kind version and to osVersion)

Reply:
Not sure but keeping it simple seems fine. In most issues those fields are not really filled out although we could enforce it here now (but then it might not make sense in all cases...)

Reply:
+1 keeping it simple as well, though these 3 (k8s, kind/minikube/os versions) were in the current template as well so I kept them. Sometimes it might be useful to know at least k8s version at least? I think we should make them as optional and leave to the user to fill it out or leave.

Reply:
I have kept the k8s version and made it optional, so if user reporting wants to report it, they can, otherwise left empty. Please let me know if we need to remove it completely, since I should admit there are different contexts for the k8s version